src/Diagnostics.HealthChecks/Readme.md

```diff
@@ -0,0 +1,105 @@
+## Omex Health Checks
+
+This project and the [`Diagnostics.HealthChecks.AspNetCore`](../Diagnostics.HealthChecks.AspNetCore/Readme.md) offers a range of composable classes, all of which implements the .NET `IHealthCheck` interface, to help the developer build their own Health Checks by reusing different behaviours with the help of extension methods.
+
+This approach enables the ability to create new composable objects to enrich the single Health Check with different behaviour, or substitute an existing one in the composition chain, offering more flexibility to the
```

Comment:
flexibility to the what?
*the sentence does not appear to be complete.

Reply:
Thanks for spotting that, solved.